sdk/python/feast/feature_store.py

```diff
@@ -58,6 +68,9 @@ def __init__(
     def _get_provider(self) -> Provider:
         return get_provider(self.config)
 
+    def _get_offline_store(self) -> Type[OfflineStore]:
```

Comment:
Do we need a method? Any reason we can't use the function directly?

Reply:
we need a method to specify which implementation of OfflineStore we're using

Reply:
I mean can we just call `get_offline_store(self.config)` and remove some of these private methods?